rsconf/component/__init__.py

```diff
@@ -162,6 +163,14 @@ def install_access(self, mode=None, owner=None, group=None):
             ),
         )
 
+    @contextlib.contextmanager
+    def install_access_temp(self, **kwargs):
```

Comment:
I think "temp" is not a great name. Rather, if you want this feature, just add "mode" to install_resource and have it do the temporary setting of the access.

Reply:
I agree on the name. I left a review on this code asking for your opinion but forgot to hit submit. Anyways, I was thinking if you were ok with this pattern then I would try to find a way to make install_resource work like open. That is, it could be called as a "normal" function or as a contexmanager. In the normal function case it would do what it does now. In the contextmanager case it would do what this function is doing. Would you prefer that or adding mode to install_resource?

Reply:
install_access is designed to support "grouping" easily in the install process. The idea is that most installs have a common pattern: resources are read only, directories are writable, etc. I don't think there is a need for context. In this specific case, if you move the [`install_access`](https://github.com/radiasoft/rsconf/pull/233/files#diff-b972143365867703eab8e161882070dc75a75c0547d7f5744cc8281cd7dc7fa8R63) inside the for loop, you don't need a temporary change to the install_access.

Reply:
9032a86